internal/harnesses/container/container.go

```diff
@@ -87,18 +90,27 @@ func New(_ context.Context, name string, cfg Config) (types.Harness, error) {
 		})
 	}
 
+	if cfg.Command == nil || len(cfg.Command) == 0 {
```

Comment:
```suggestion
	if len(cfg.Command) == 0 {
```

this should be enough, len(nil) == 0